pkgs/tools/security/oauth2l/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoPackage, fetchFromGitHub, ... }:
+
+let version = "1.2.2"; in
+
+buildGoPackage {
```

Comment:
Oh I'm sorry, I missed this. Can you change to `buildGoModule`? The nixpkgs documentation says `buildGoPackage` is the legacy way:
https://nixos.org/manual/nixpkgs/stable/#sec-language-go

Reply:
oops! sure, thanks for the catch- this absolutely was just a template pulled from the closest simplest go build.

thanks for paying attention!

Reply:
having some issue with the test environment, will circle back when i've figured it out.

Reply:
got it- had to stick in a bummer of a flag, but that seems the only fix to be made without modifying the source.